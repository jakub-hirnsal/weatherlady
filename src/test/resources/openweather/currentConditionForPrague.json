{
  "coord": {
    "lon": 14.4208,
    "lat": 50.088
  },
  "weather": [
    {
      "id": 803,
      "main": "Clouds",
      "description": "broken clouds",
      "icon": "04d"
    }
  ],
  "base": "stations",
  "main": {
    "temp": 4.28,
    "feels_like": 1.99,
    "temp_min": 3.5,
    "temp_max": 7.37,
    "pressure": 1018,
    "humidity": 95
  },
  "visibility": 10000,
  "wind": {
    "speed": 2.57,
    "deg": 200
  },
  "clouds": {
    "all": 75
  },
  "dt": 1634457331,
  "sys": {
    "type": 2,
    "id": 20854,
    "country": "CZ",
    "sunrise": 1634448441,
    "sunset": 1634486857
  },
  "timezone": 7200,
  "id": 3067696,
  "name": "Prague",
  "cod": 200
}