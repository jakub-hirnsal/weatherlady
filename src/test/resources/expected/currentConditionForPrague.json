{
  "city": "Prague",
  "countryCode": "CZ",
  "Temperature": {
    "Metric": {
      "Value": 4.28,
      "Unit": "C"
    },
    "Imperial": null
  },
  "Wind": {
    "Direction": {
      "Degrees": 200,
      "Localized": null
    },
    "Speed": {
      "Metric": {
        "Value": 2.57,
        "Unit": "??"
      },
      "Imperial": null
    }
  },
  "Pressure": {
    "Metric": {
      "Value": 1018.0,
      "Unit": "mb"
    },
    "Imperial": null
  }
}